examples/README.md

```diff
@@ -72,6 +72,14 @@ neurodocker generate [docker|singularity] --base=debian:stretch --pkg-manager=ap
   --install git vim
 ```
 
+by default --install uses --no-install-recommends to minimize container sizes. In few cases this can lead to unexpected behaviours and one can try to build a container without this option.
```

Comment:
can you clarify the syntax for `--install` here? you can use `apt_opts` to set options for `apt-get install` and `yum_opts` to set any options for `yum install`. so if you wanted something in addition to `--no-install-recommends`, you would use `apt_opts='--no-install-recommends --no-upgrade'. it would be great if you could clarify this for other users because at the moment, this feature is undocumented.